src/main/groovy/com/rundeck/plugins/ansible/ansible/AnsibleDescribable.java

```diff
@@ -527,4 +529,13 @@ public static String[] getValues() {
             .title("Encrypt Extra Vars.")
             .description("Encrypt the value of the extra vars keys.")
             .build();
+
+    Property YAML_DATA_SIZE_PROP = PropertyBuilder.builder()
+            .string(ANSIBLE_YAML_DATA_SIZE)
+            .required(false)
+            .title("Inventory Yaml Data Size")
+            .description("Set the MB size (Default value is 10MB)"+
```

Comment:
```suggestion
            .description("Set the MB size (Default value is 10)"+
```
I think this way is more clear that it's only required to put the value and not the unit

Reply:
Done... ✅